_includes/nav-desktop.html

```diff
@@ -20,7 +20,7 @@
 			<li class="nav__item"><a href="{{ basenav }}/merchants/" class="nav__link">{% t nav.merchants %}</a></li>
 			<li class="nav__item"><a href="{{ basenav }}/network/" class="nav__link">{% t nav.currency %}</a></li>
 			<li class="nav__item"><a href="{{ basenav }}/community/" class="nav__link">{% t nav.community %}</a></li>
-            <li class="nav__item"><a href="{{ basenav }}/team" class="nav__link">{% t nav.team %}</a></li>
+            <li class="nav__item"><a href="{{ basenav }}/team/" class="nav__link">{% t nav.team %}</a></li>
```

Comment:
indent is missing

Reply:
Indent was missing previously (check the diff), I only changed the `/`.